.github/workflows/phpunit-tests.yml

```diff
@@ -154,13 +154,45 @@ jobs:
       phpunit-config: ${{ matrix.multisite && 'tests/phpunit/multisite.xml' || 'phpunit.xml.dist' }}
       report: ${{ matrix.report || false }}
 
+  #
+  # Runs specific test groups.
+  #
+  specific-test-groups:
+    name: PHP ${{ matrix.php }}
+    uses: WordPress/wordpress-develop/.github/workflows/reusable-phpunit-tests-v3.yml@trunk
+    permissions:
+      contents: read
+    secrets: inherit
+    if: ${{ github.repository == 'WordPress/wordpress-develop' || github.event_name == 'pull_request' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest ]
+        php: [ '8.3' ]
+        db-type: [ 'mysql' ]
+        db-version: [ '8.0' ]
+        multisite: [ false ]
+        memcached: [ false ]
+        phpunit-test-groups: [ 'html-api-html5lib-tests' ]
+
+    with:
+      os: ${{ matrix.os }}
+      php: ${{ matrix.php }}
+      db-type: ${{ matrix.db-type }}
+      db-version: ${{ matrix.db-version }}
+      multisite: ${{ matrix.multisite }}
+      memcached: ${{ matrix.memcached }}
+      phpunit-config: ${{ matrix.multisite && 'tests/phpunit/multisite.xml' || 'phpunit.xml.dist' }}
+      phpunit-test-groups: ${{ matrix.phpunit-test-groups }}
+      report: ${{ matrix.report || false }}
```

Comment:
Why not report? I believe this flag is for this:

> Submit test results to the WordPress.org host test results

I don't know what that implies or means exactly, but these tests should really have the same weight as any other phpunit tests. They should not fail on anything merged for release.

Reply:
This tells the workflow to submit the results of running the test suite to the [w.org hosting tests](https://make.wordpress.org/hosting/test-results/).

Currently, the hosting tests does support submitting multiple reports per commit, but it does not support submitting subsets of tests. The assumption is that you run the full test suite, there would be [no indication that a report only represents a small segment](https://make.wordpress.org/hosting/test-results/r59526/bluehostbot-r59526-8-1-mysql-ver-14-14-distrib-5-7-23-23-for-linux-x86_64-using-6-2/).

I think, if we wanted to include the results of these tests when submitting, the reusable workflow would need to be updated to somehow include the provided test group and not only group excluding the standard `phpunit.xml` setup. This could still run on just one job, but it would then again become nested under the PHP version where it was included.

Let's start without including it in a report and we can always look to include it in the future.

Reply:
👍 Sure, that sounds good. Just getting these tests running on CIis a big benefit.